include/envoy/http/async_client.h

```diff
@@ -220,15 +220,21 @@ class AsyncClient {
       parent_span_ = &parent_span;
       return *this;
     }
+    RequestOptions& setChildSpanName(const std::string& child_span_name) {
+      child_span_name_ = child_span_name;
+      return *this;
+    }
 
     // For gmock test
     bool operator==(const RequestOptions& src) const {
-      return StreamOptions::operator==(src) && parent_span_ == src.parent_span_;
+      return StreamOptions::operator==(src) && parent_span_ == src.parent_span_ &&
+             child_span_name_ == src.child_span_name_;
     }
 
     // The parent span that child spans are created under to trace egress requests/responses.
     // If not set, requests will not be traced.
     Tracing::Span* parent_span_{nullptr};
+    std::string child_span_name_{""};
```

Comment:
Mind adding a comment, like the one for parent_span_ above?  Probably worth mentioning if this is not set (and parent tracing is non-null) there will be a default value set.

Reply:
Sure! Thanks